Packages/Core/Runtime/Variables/AtomVariable.cs

```diff
@@ -47,6 +47,16 @@ public abstract class AtomVariable<T, P, E1, E2, F> : AtomBaseVariable<T>, IGetE
         /// </summary>
         public E2 ChangedWithHistory;
 
+        /// <summary>
+        /// Whether Changed Event should be triggered on OnEnable or not
+        /// </summary>
+        public bool TriggerChangedOnOnEnable;
```

Comment:
Make the variable private, rename it to `_triggerChangedOnOnEnable ` and add the attribute `[SerializeField]`.

Reply:
Forgot initially to add that for clarity sake you should also explicitly set the value to `default` or `false`.